npe2/manifest/contributions/_menus.py

```diff
@@ -1,27 +1,26 @@
+from collections import namedtuple
 from typing import Optional, Union
 
 from pydantic import BaseModel, Field
 
 from ..utils import Executable
 
-# # napari provides these
-# class Menu(BaseModel):
-#     key: str
-#     id: int
-#     description: str
-#     supports_submenus: bool = True
-#     deprecation_message: Optional[str]
-
-
-# napari_menus = [
-#     Menu(key="command_pallete", id=0, description="The Command Palette"),
-#     Menu(
-#         key="layers__context", id=1, description="The layer list context menu"
-#     ),
-#     Menu(
-#         key="layers__context", id=1, description="The layer list context menu"
-#     ),
-# ]
+Menu = namedtuple("Menu", "key description")
```

Comment:
@kne42 should we make the "id, description" - trying to match what you did here https://github.com/napari/napari/pull/4497/files#diff-9ea32b5da7975500739c6eb2f806a3ca84c916c445cf4e5051b6e9d5fe903312R8